packages/blockchain-wallet-v4-frontend/src/scenes/Login/selectors.ts

```diff
@@ -0,0 +1,37 @@
+import { lift } from 'ramda'
+
+import { ExtractSuccess } from '@core/types'
+import { selectors } from 'data'
+import { RootState } from 'data/rootReducer'
+import { LoginFormType, LoginSteps } from 'data/types'
+
+import { LOGIN_FORM_NAME } from './model'
```

Comment:
can we use the constant from `data` so we dont have this in two places? i.e. i think its `data.auth.model.LOGIN_FORM`